instance_clusters.go

```diff
@@ -0,0 +1,206 @@
+func (s *InstanceClustersService) GetCluster(clusterID int, options ...RequestOptionFunc) (*InstanceCluster, *Response, error) {
+	u := fmt.Sprintf("admin/clusters/%d", clusterID)
+
+	req, err := s.client.NewRequest("GET", u, nil, options)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	ic := new(InstanceCluster)
+	resp, err := s.client.Do(req, &ic)
+	if err != nil {
+		return nil, resp, err
+	}
+
+	return ic, resp, err
+}
+
+// AddInstanceClusterOptions represents the available AddCluster() options.
+//
+// GitLab API docs:
+// https://docs.gitlab.com/ee/api/instance_clusters.html#add-existing-cluster-to-instance
+type AddInstanceClusterOptions struct {
+	Name                *string                               `url:"name,omitempty" json:"name,omitempty"`
+	Domain              *string                               `url:"domain,omitempty" json:"domain,omitempty"`
+	Enabled             *bool                                 `url:"enabled,omitempty" json:"enabled,omitempty"`
+	Managed             *bool                                 `url:"managed,omitempty" json:"managed,omitempty"`
+	EnvironmentScope    *string                               `url:"environment_scope,omitempty" json:"environment_scope,omitempty"`
+	PlatformKubernetes  *AddInstancePlatformKubernetesOptions `url:"platform_kubernetes_attributes,omitempty" json:"platform_kubernetes_attributes,omitempty"`
+	ManagementProjectID *string                               `url:"management_project_id,omitempty" json:"management_project_id,omitempty"`
+}
+
+// AddInstancePlatformKubernetesOptions represents the available PlatformKubernetes options for adding.
+type AddInstancePlatformKubernetesOptions struct {
+	APIURL            *string `url:"api_url,omitempty" json:"api_url,omitempty"`
+	Token             *string `url:"token,omitempty" json:"token,omitempty"`
+	CaCert            *string `url:"ca_cert,omitempty" json:"ca_cert,omitempty"`
+	Namespace         *string `url:"namespace,omitempty" json:"namespace,omitempty"`
+	AuthorizationType *string `url:"authorization_type,omitempty" json:"authorization_type,omitempty"`
+}
+
+// AddCluster adds an existing cluster to the instance.
+//
+// GitLab API docs:
+// https://docs.gitlab.com/ee/api/instance_clusters.html#add-existing-instance-cluster
+func (s *InstanceClustersService) AddCluster(opt *AddInstanceClusterOptions, options ...RequestOptionFunc) (*InstanceCluster, *Response, error) {
+	u := fmt.Sprintf("admin/clusters/add")
+
+	req, err := s.client.NewRequest("POST", u, opt, options)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	ic := new(InstanceCluster)
+	resp, err := s.client.Do(req, ic)
+	if err != nil {
+		return nil, resp, err
+	}
+
+	return ic, resp, err
+}
+
+// EditInstanceClusterOptions represents the available EditCluster() options.
+//
+// GitLab API docs:
+// https://docs.gitlab.com/ee/api/instance_clusters.html#edit-instance-cluster
+type EditInstanceClusterOptions struct {
```

Comment:
```suggestion
type EditClusterOptions struct {
```

Reply:
Same thing here, `EditClusterOptions` is already declared in `https://github.com/xanzy/go-gitlab/blob/d3946d4f8c62bfae945062be289f13ea5e6ecfe4/project_clusters.go#L177`